subgraph/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@rainprotocol/orderbook-subgraph",
+  "description": "Rain OrderBook Subgraph",
+  "version": "0.1.0",
+  "author": "rainprotocol",
+  "license": "CAL",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build"
+  },
+  "dependencies": {
+    "@graphprotocol/graph-cli": "0.46.1",
```

Comment:
can we pin our deps?

Reply:
Well, that way also is for pinning deps, always will resolve to that specific version. But I used the `=` to make it explicit.